itou/templates/approvals/includes/list_card.html

```diff
@@ -2,38 +2,76 @@
-                    PASS IAE {{ approval.get_state_display|lower }}
-                </span>
-            </div>
-        </div>
-
-        <div class="fs-sm">
-            Nombre de jours restants sur le PASS IAE :
-            {% if approval.is_valid %}
-                {{ approval.remainder.days }} jour{{ approval.remainder.days|pluralizefr }}
-            {% else %}
-                0 jour
-            {% endif %}
-            <i class="ri-information-line ri-xl text-info"
-               data-bs-toggle="tooltip"
-               title="Le reliquat est calculé sur la base d'un nombre de jours calendaires. Si le PASS IAE n'est pas suspendu, il décroît donc tous les jours (samedi, dimanche et jours fériés compris)."></i>
+	
         </div>
+	
     </div>
-
-    <div class="card-footer text-end bg-light">
-        <a href="{% url 'approvals:detail' pk=approval.id %}"
-           class="btn btn-sm btn-outline-primary bg-white"
-           {% matomo_event "salaries" "clic" "details-salarie" %}
-           aria-label="Voir le profil de {{ approval.user.get_full_name }}">Voir son profil</a>
+	
+    <hr class="m-0">
+	
+    <div class="c-box--results__body">
+	
+        <ul class="list-data list-data__two-column-md mb-3">
+	
+            <li>
+                <small>Numéro de PASS IAE</small>
+                <strong>{{ approval|format_approval_number }}</strong>
+	
+                <button class="btn-link" data-bs-toggle="tooltip" data-bs-placement="top" data-bs-trigger="manual" data-bs-title="Copié!" data-it-clipboard-button="copy" data-it-copy-to-clipboard="{{ approval }}">
+                    <i class="ri-file-copy-line"></i>
+                </button>
+	
+            </li>
+	
+            <li>
+                <small>
+                    Durée de validité
+                    <i class="ri-information-line text-info"
+                       data-bs-toggle="tooltip"
+                       title="Le reliquat est calculé sur la base d'un nombre de jours calendaires. Si le PASS IAE n'est pas suspendu, il décroît donc tous les jours (samedi, dimanche et jours fériés compris)."></i></small>
+                <strong class="text-success">
+                    {% if approval.is_valid %}
+                        {{ approval.remainder.days }} jour{{ approval.remainder.days|pluralizefr }}
+                    {% else %}
+                        0 jour
+                    {% endif %}
+                </strong>
+	
+            </li>
+	
+        </ul>
+	
+    </div>
+	
+    <div class="c-box--results__footer">
+	
+        <div class="d-flex justify-content-md-end">
+            <a href="{% url 'approvals:detail' pk=approval.id %}"
+               class="btn btn-outline-primary btn-block w-100 w-md-auto"
+               {% matomo_event "salaries" "clic" "details-salarie" %}
+               aria-label="Voir le profil de {{ approval.user.get_full_name }}">Voir la fiche salarié</a>
```

Comment:
Fiche salarié est déjà pris. Peut-être aussi synchroniser la terminologie du `aria-label` avec le texte affiché ?
```suggestion
               aria-label="Voir le profil de {{ approval.user.get_full_name }}">Voir les informations</a>
```